### PR TITLE
[AAE-10214] Storybook ErrorContent Component

### PR DESCRIPTION
**Pull request by marcol13**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**
Added three stories for ErrorContent Component:
- **Error Code Story**

Handle input codeError changes

image

- **Error Param with Known ID**

Shows known error caused by Route (500 error)

image

- **Error Param with Unknown ID**

Shows unknown error caused by Route (200 error)

image

Video:

https://user-images.githubusercontent.com/56632321/183901739-a4578f5c-98f5-476d-ad4d-7157cdb77514.mp4

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: lib/core/templates/error-content/error-content.component.stories.ts [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+import { ActivatedRoute } from '@angular/router';
+import { MatButtonModule } from '@angular/material/button';
+import { of } from 'rxjs';
+
+export default {
+    component: ErrorContentComponent,
+    title: 'Core/Template/Error Content',
+    decorators: [
+        moduleMetadata({
+            imports: [CoreStoryModule, TemplateModule, MatButtonModule],
```
Comment (tomgny) [inline lib/core/templates/error-content/error-content.component.stories.ts:31]: can we move matButtonModule to Template module, and remove MaterialModule from it?

File: lib/core/templates/error-content/error-content.component.stories.ts [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+                },
+                defaultValue: {
+                    summary: false
+                }
+            }
+        }
+    }
+} as Meta;
+
+export const errorCodeStory: Story = args => ({
```
Comment (tomgny) [inline lib/core/templates/error-content/error-content.component.stories.ts:86]: can we move that story below templates to improve stories readability?

**Review by tomgny — CHANGES_REQUESTED** [review]
Please check comments

File: lib/core/templates/error-content/error-content.component.stories.ts [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+const templateArgTypes = {
+    errorCode: {
+        control: {
+            disable: true
+        }
+    }
+};
+
+const template: Story = args => ({
```
Comment (tomgny) [inline lib/core/templates/error-content/error-content.component.stories.ts:104]: can we add type for args and Story? 
```suggestion
const template: Story<ErrorContentComponent> = (args: ErrorContentComponent) => ({
```

File: lib/core/templates/error-content/error-content.component.stories.ts (hunk 1 above)
Comment (marcol13) [reply lib/core/templates/error-content/error-content.component.stories.ts:31]: Done

File: lib/core/templates/error-content/error-content.component.stories.ts (hunk 2 above)
Comment (marcol13) [reply lib/core/templates/error-content/error-content.component.stories.ts:86]: Done

File: lib/core/templates/error-content/error-content.component.stories.ts (hunk 3 above)
Comment (marcol13) [reply lib/core/templates/error-content/error-content.component.stories.ts:104]: Done

**Review by tomgny — APPROVED** [review]

**Pull request merged**